src/reducers/cities-reducer.js

```diff
@@ -0,0 +1,44 @@
+import { FETCH_FORECAST } from "../actions/index";
+
+const defaultState = [];
+
+const citiesReducer = (state=defaultState, action) => {
+  //Checks to see the fetch request failed and returns the current state if so
+  if(action.error) {
+    alert('Could not find results for search')
+    return state;
+  }
+  // (temperature in Kelvin − 273.15) × 9/5 + 32 = temperature in Fahrenheit
+  const convertKelvinToFahrenheit = (temperatureArr) => temperatureArr.map((t) => Math.round((t - 273.15) * (9/5) + 32)); 
+
+  // Grabs the specified weather data type froom the openweather API results
+  const getWeatherDataFromFetch = (openWeatherData, type) => {
+    return openWeatherData.data.list.map((forecastPoint) => forecastPoint.main[type])
+  };
+
+  //Grabs the times of the forecast points
+  const getTimesOfForecastPoints = (openWeatherData) => {
```

Comment:
In the future you'd probably want to be helpers somewhere else just to make this reducer more readable.